PurchasesCoreSwift/Logging/LoggerWrapper.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  LoggerWrapper.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/08/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCLogIntent) public enum LogIntent: Int {
+    case appleError
+    case info
+    case purchase
+    case rcError
+    case rcPurchaseSuccess
+    case rcSuccess
+    case user
+    case warning
+    
+    func emojis() -> String {
+        switch self {
+        case .appleError: return Emojis.appleError.rawValue + Emojis.doubleExclamation.rawValue
+        case .info: return Emojis.info.rawValue
+        case .purchase: return Emojis.purchase.rawValue
+        case .rcError: return Emojis.rcError.rawValue + Emojis.doubleExclamation.rawValue
+        case .rcPurchaseSuccess: return Emojis.rcSuccess.rawValue + Emojis.purchase.rawValue
+        case .rcSuccess: return Emojis.rcSuccess.rawValue
+        case .user: return Emojis.appUserID.rawValue
+        case .warning: return Emojis.warning.rawValue
+        }
+    }
+}
+
+@objc(RCLoggerWrapper) public class LoggerWrapper: NSObject {
+    @objc public static func log(level: LogLevel, intent: LogIntent, message: String) {
+        let emojifiedMessage = "\(intent.emojis()) \(message)"
+        Logger.log(level: level, message: emojifiedMessage)
+    }
+    @objc public static func appleError(_ message: String) {
+        log(level: .error, intent: .appleError, message: message)
+    }
+    
+    @objc public static func appleInfo(_ message: String) {
```

Comment:
Following Android implementation, this is for logs that not necessarily an error, but the source is from Apple. Example: "could not find SKProduct for..." and "App running in sandbox without a receipt file. Restoring transactions won't work..."

Reply:
maybe those should be of warning level?

Reply:
since we want developers to know the source of the error, would this just be another LogIntent case with `appleWarn = apple + warning` or cluster these logs with `warning`?

Reply:
I meant doing something like `log.appleWarning("skus can't be found")`, so that we have a way for them to stand out in the logs